tests/test_pydantic/test_field.py

```diff
@@ -22,16 +22,6 @@ class Model(SQLModel):
         Model(dec=Decimal("420"))
 
 
-def test_unique_items():
-    class Model(SQLModel):
-        unique_strings: List[str] = Field(unique_items=True)
-
-    Model(unique_strings=["x", "y"])
-
-    with pytest.raises(ValidationError):
-        Model(unique_strings=["x", "y", "x"])
-
-
```

Comment:
why remove and not skipif, as pydantic v2 support is not present now? 

Please note that if that means Pydantic v2 support in few days, I'm fine with it ;-)

Reply:
Yep, the plan is to support SQLAlchemy 2.0 and Pydantic 2.0 as soon as possible. 🤓